nvme.c

```diff
@@ -5317,6 +5321,330 @@ static int show_registers(int argc, char **argv, struct command *cmd, struct plu
 	return err;
 }
 
+bool nvme_is_fabrics_reg(int offset)
+{
+	switch (offset) {
+	case NVME_REG_CAP:
+		fallthrough;
+	case NVME_REG_VS:
+		fallthrough;
+	case NVME_REG_CC:
+		fallthrough;
+	case NVME_REG_CSTS:
+		fallthrough;
+	case NVME_REG_NSSR:
+		fallthrough;
```

Comment:
`fallthrough` is unusual for cases that don't have any body. I have typically seen all the cases listed adjacently:
```c
case NVME_REG_CAP:
case NVME_REG_VS:
case NVME_REG_CC:
case NVME_REG_CSTS:
case NVME_REG_NSSR:
case NVME_REG_CRTO:
        return true;
default:
        return false;
```

Reply:
I could understand then fixed by the commit 49862bf5. Thanks for your explanation.